.github/workflows/plugin-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Plugin-Release
+
+on: pull_request
+
+jobs:
+
+  ubuntu:
```

Comment:
The job should have a different name, e.g. 'plugin'

Reply:
Addressed in f17112c